src/swift_server.rs

```diff
@@ -424,13 +467,10 @@ pub async fn start_server() {
         "mainnet-beta" => Context::MainNet,
         _ => panic!("Invalid drift environment: {drift_env}"),
     };
-    let client = DriftClient::new(
-        context,
-        RpcClient::new(rpc_endpoint),
-        Keypair::new().into(), // not sending txs
-    )
-    .await
-    .expect("initialized client");
+    let wallet = Wallet::new(keypair.unwrap());
+    let client = DriftClient::new(context, RpcClient::new(rpc_endpoint), wallet)
```

Comment:
nit: then won't need the copy on line 458
```suggestion
    let client = DriftClient::new(context, RpcClient::new(rpc_endpoint.clone()), wallet)
```